pkg/rollout/rollout.go

```diff
@@ -114,7 +122,7 @@ func (r *Rollout) UpdateService(svc *run.Service) (*run.Service, error) {
 	if isNewCandidate(svc, candidate) {
 		r.log.Debug("new candidate, assign some traffic")
 		svc = r.PrepareRollForward(svc, stable, candidate)
-		svc = r.updateAnnotations(svc, stable, candidate)
+		svc = r.updateAnnotations(svc, stable, candidate, "new candidate, no health report available yet")
```

Comment:
this is not a great way. I'll offer alternatives.

1.  pass a `statusMsgFunc func()string` that returns the msg you wanna set.
2. create a new method `svc = r.setAnnotation(key, msg)`